### PR TITLE
Add project-discovery katana tool.

### PR DESCRIPTION
**Pull request by Fisjkars**

### CODE REVIEW

**Comment by ShutdownRepo** [comment]
Hi @Fisjkars! Thank you for those additions! The problem is we are in the middle of important changes and refactoring. We will be releasing those changes very soon and you should then be able to update your PR accordingly.
Briefly put, the only additional thing needed here will be a test command that our new CI/CD pipeline will be able to run to make sure that katana was installed correctly, something like `tool --help`.

**Comment by Fisjkars** [comment]
Ok cool , looking forward to see the changes. Could you, please, poke me when I will be able to update the PR?  

Thanks in advance !

File: sources/install.sh [hunk 1]
```diff
@@ -1928,6 +1928,12 @@ function install_nuclei() {
   add-history nuclei
 }
 
+function install_katana() {
+  # Crawling and Spidering
```
Comment (QU35T-code) [inline sources/install.sh:1932]: I think this comment is not necessary (as it is present in the package_web)

File: sources/zsh/history [hunk 2]
```diff
@@ -1,2 +1,2 @@
 mount -o domain="$DOMAIN" -o username="$USER" -o password="$PASSWORD" -t cifs "//$DC_HOST/SYSVOL" /tmp/mnttarget/
-screen /dev/ttyACM0 115200
\ No newline at end of file
+screen /dev/ttyACM0 115200
```
Comment (QU35T-code) [inline sources/zsh/history:2]: There has been a lot of change with the management of `history` and `aliases`. Now, history and aliases are separated by tools.
The folder `sources/zsh/history.d/` contains history files and the folder `sources/zsh/aliases.d/` contains aliases files.
If you want to add a `history command` as I recommended, you have to create a file with the name of the tool in the `history.d` folder.
You can take an example from : https://github.com/ThePorgs/Exegol-images/blob/main/sources/zsh/history.d/assetfinder.

**Comment by Dramelac** [comment]
Hello @Fisjkars 

You can update your PR from the dev branch, the new CI/CD pipeline is ready!
Once done with the integration of @QU35T-code comments we will run the test pipeline and merge your PR

Thank you for your contribution :pray:

**Comment by ShutdownRepo** [comment]
Hello @Fisjkars, it seems there are some conflicts. Can you update/resolve them so that I can run the checks and merge the PR once validated?

**Comment by Fisjkars** [comment]
Hello @ShutdownRepo, sorry for the late contribution.

I prefered to open a fresh new merge request (https://github.com/ThePorgs/Exegol-images/pull/128) instead of struggling with conflicts :) 

+++++

**Pull request closed without merging**